docs/modules/ROOT/partials/getting-started/ref-registry-security-configuration.adoc

```diff
@@ -5,16 +5,18 @@
 = {registry} authentication and authorization configuration options
 
 [role="_abstract"]
-This section describes the authentication and authorization options for {registry} using {keycloak}. 
+This section describes the authentication and authorization options for {registry} using {keycloak}.
 
 You can enable authentication for the {registry} web console and core REST API using {keycloak}. The same {keycloak} realm and users are federated across the {registry} web console and core REST API using Open ID Connect (OIDC) so that you only require one set of credentials.
 
-{registry} provides role-based authorization for default admin, write, and read-only user roles. {registry} also provides content-based authorization at the schema or API level, where only the creator of the registry artifact can update or delete it. {registry} authentication and authorization options are disabled by default. 
+{registry} supports a number of authorization configurations, including role-based and content-based approaches.
+{registry} provides role-based authorization for default admin, write, and read-only user roles.
+{registry} also alternatively provides content-based authorization at the schema or API level, where only the creator of the registry artifact can update or delete it. {registry} authentication and authorization options are disabled by default.
```

Comment:
Can you configure both role-based authz and content-based authz at the same time? 

"{registry} also alternatively provides content-based authorization" suggests you cannot.  Suggest dropping the "alternatively" if you can?

Reply:
Yes you can.  Change accepted. :)